target/linux/octeon/image/Makefile

```diff
@@ -29,6 +29,16 @@ define Device/generic
 endef
 TARGET_DEVICES += generic
 
+ITUSROUTER_CMDLINE:=console=ttyS0,115200, root=/dev/mmcblk1p2 rootfstype=squashfs,ext4,f2fs rootwait
+define Device/itus,shield-router
```

Comment:
The device definition should be kept with underscore, _in contrast_ to the board name:
itus_shield-router

Reply:
Will correct